server/src/main/java/org/candlepin/bind/ToDo.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2009 - 2012 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.bind;
+
+import org.candlepin.controller.PoolManager;
+import org.candlepin.model.Consumer;
+import org.candlepin.model.Entitlement;
+import org.candlepin.model.PoolQuantity;
+
+import com.google.inject.Inject;
+
+import java.util.Map;
+
+/**
+ * Created by vrjain on 5/5/17.
+ */
+public class ToDo implements BindOperation {
```

Comment:
@barnabycourt that was work in progress, to indicate the operations that will be tackled in the next PR.